include/antidote.hrl

```diff
@@ -111,14 +111,17 @@
 	  tx_id :: txid(),
 	  op_type :: update | prepare | commit | abort | noop,
 	  log_payload :: #commit_log_payload{}| #update_log_payload{} | #abort_log_payload{} | #prepare_log_payload{}}).
--record(op_number, {node :: {node(),dcid()}, global :: non_neg_integer(), local :: non_neg_integer()}).
+-record(op_number, {
+  node :: undefined | {node(),dcid()},
+  global :: undefined | non_neg_integer(),
+  local :: undefined | non_neg_integer()}).
 
 %% The way records are stored in the log.
 -record(log_record,{
 	  version :: non_neg_integer(), %% The version of the log record, for backwards compatability
-	  op_number :: #op_number{},
-	  bucket_op_number :: #op_number{},
-	  log_operation :: #log_operation{}}).
+	  op_number :: undefined | #op_number{},
+	  bucket_op_number :: undefined | #op_number{},
```

Comment:
Why precisely would these ever be undefined? How about instead of modifying the type signature, we identify the location in the code where they are being initialized to undefined and change that to use a starting value.

What do you think?